scene/3d/label_3d.h

```diff
@@ -253,7 +253,7 @@ class Label3D : public GeometryInstance3D {
 	StandardMaterial3D::TextureFilter get_texture_filter() const;
 
 	virtual AABB get_aabb() const override;
-	Ref<TriangleMesh> generate_triangle_mesh() const;
+	Ref<TriangleMesh> generate_triangle_mesh() const override;
```

Comment:
```suggestion
	virtual Ref<TriangleMesh> generate_triangle_mesh() const override;
```
This *should* have triggered a build error, unsure why it didn't

Reply:
Good eye as usual :)
Done!